IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CableDuct/DocPropertySet.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CableDuct" UniqueId="6f8bdb97-e9de-e4c8-d8be-75ad71cc6cfa" ApplicableType="IfcDistributionChamberElement/FORMEDDUCT" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
I have erased the Pset and associated the property to the pset "Pset_DistributionChamberElementTypeFormedDuct". Commit : 0ff39677556020cb6ef596d6fc991b3c9c545a0f